app/views/mailers/conversation_reply_mailer/conversation_transcript.html.erb

```diff
@@ -14,6 +14,9 @@
           Attachment [<a href="<%= attachment.file_url %>" _target="blank">Click here to view</a>]
         <% end %>
       <% end %>
+      <p style="font-size: 90%; color: #808080;">
+        <%= message.created_at.in_time_zone(@inbox.timezone).strftime('%b %d, %Y, %I:%M %p') if message.created_at %>
```

Comment:
@CristianDuta inbox.timezone is not a mandatory field. Any idea how to navigate that?

Reply:
@pranavrajs Thank you for your question. 

That is indeed true and I thought of it. Although the timezone is not mandatory, a default value of UTC is set for each inbox channel type.

I also believe it is the duty of the administrator to set a timezone for the inboxes.

That being said, we could also add the TZ at the end to be more precise. What do you think?

Reply:
@pranavrajs would you like to see the timezone at the end of the string or do you think it would be too much? 

e.g.: `Aug 2, 2023, 07:26 AM UTC` or `Aug 2, 2023, 09:26 AM Europe/Berlin`

Reply:
Sounds good. I've updated the logic. However, I've removed the year from the timestamp. It doesn't seem necessary, but I can add it again if needed after feedback from customers.

Reply:
![Uploading Screenshot 2023-08-04 at 5.16.06 PM.png…]()

Reply:
@pranavrajs Thank you! Cheers!